src/PostgrestFilterBuilder.ts

```diff
@@ -599,4 +602,28 @@ export default class PostgrestFilterBuilder<
     this.url.searchParams.append(column, `${operator}.${value}`)
     return this
   }
+
+  /**
+   * Set the maximum number of rows that can be affected by the query.
+   * Only available in PostgREST v13+ and only works with PATCH and DELETE methods.
+   *
+   * @param value - The maximum number of rows that can be affected
+   */
+  maxAffected(
```

Comment:
Let's move this to `PostgrestTransformBuilder` since this doesn't do horizontal filtering, but rather an assertion a la `.single()`